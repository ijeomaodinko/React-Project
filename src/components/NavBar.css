nav {
display: flex;
flex-direction: row;
background-color: #1f255e ;
height: 4rem;
justify-content: space-between;
color: #efeaf3;
width:96%;
position: relative;
align-items: center;
margin-left: auto;
margin-right: auto;
padding-left: 1.9rem;
}

h1, h5 {
color: #e2eaf3;
text-decoration: none;
list-style: none;
}

.img {
    width: 25%;
}