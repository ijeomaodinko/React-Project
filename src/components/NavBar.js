import { NavLink } from 'react-router-dom';
import item from './assets/item.jpeg';
import itemi from './assets/itemi.png';

const NavBar = () => (
  <nav>
    <div>
      <h1> Weather Forecast </h1>
      <NavLink to="/"><h2>HOME</h2></NavLink>
    </div>
    <div>
      <img src={item} alt="setting" />
      <img src={itemi} alt="speaker" />
    </div>
  </nav>
);

export default NavBar;
