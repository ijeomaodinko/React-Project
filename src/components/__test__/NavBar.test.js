import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import store from '../../redux/configureStore';
import { BrowserRouter } from 'react-router-dom';
import NavBar from '../NavBar'

describe('To test for the home page', () => {
  it('to match the dom snapshot', () => {
    const { tree } = render(
      <BrowserRouter>
        <Provider store={store}>
          <NavBar />
        </Provider>
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
