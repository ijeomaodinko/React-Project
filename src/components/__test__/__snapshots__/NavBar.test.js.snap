// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`To test for the home page to match the dom snapshot 1`] = `undefined`;
